### DISCUSSION TITLE
Death Trap: Lab remix for 2.0

### DISCUSSION THREAD

**Discussion by SniperGoth** [redeclipse/base#1151] [Maps and Content Showcase]
This how i believe the map "Death Trap" would look like, in a more lab-like, clean theme (sorry, no deathtrap in a temple setting for now, hahah), for 1.9/2.0 .
Made the color saturation very strong, so it has a distinct look/feel to it.
There's quite a few of volumetrics around the center of the map, but the big ones are set to have no collision and shadow of sorts, so they barely affect performance.
Im releasing this as it is right now, before i get too excited and start modifying the layout and other visual things about it, so this should play like the one from before, but with a new coat of paint.

"Day" variant, seen from the Omega base.
![20180914192047](https://user-images.githubusercontent.com/24359960/45577989-468a4d80-b855-11e8-9b87-2963977119d7.png) 
![20180914191940](https://user-images.githubusercontent.com/24359960/45577986-468a4d80-b855-11e8-8b47-2634cfa01b92.png)
![20180914192004](https://user-images.githubusercontent.com/24359960/45577987-468a4d80-b855-11e8-84da-b3034971fe6a.png)
![20180914192025](https://user-images.githubusercontent.com/24359960/45577988-468a4d80-b855-11e8-8d0d-0c2904575884.png)

"Night" variant, from the Alpha base.

![20180914192124](https://user-images.githubusercontent.com/24359960/45578009-66ba0c80-b855-11e8-935b-aba25713b3de.png)
![20180914192134](https://user-images.githubusercontent.com/24359960/45578010-66ba0c80-b855-11e8-9474-4683234ab0fe.png)
![20180914192147](https://user-images.githubusercontent.com/24359960/45578011-66ba0c80-b855-11e8-9f15-d4d44be32091.png)
![20180914192113](https://user-images.githubusercontent.com/24359960/45578012-66ba0c80-b855-11e8-8493-fb911fa85344.png)

No custom models or textures on this one.
[deathtrap-lab.zip](https://github.com/red-eclipse/world/files/2385087/deathtrap-lab.zip)

**Comment by MirceaKitsune** [comment]
I remember this map from RE 1. The 2.0 version is even more awesome, just tried it out with the latest GIT version: Same fun gameplay and it also looks lovely now! I saw no missing textures and even bots play well without getting themselves killed.

Is there a reason why this wasn't included yet? The map looks so finished I'm thinking it must be an oversight and it just got forgotten.
